Makefile

```diff
@@ -24,9 +24,10 @@ release:
 
 # Get the build dependencies
 build_dep:
-	go get -u github.com/kisielk/errcheck
-	go get -u golang.org/x/tools/cmd/goimports
-	go get -u github.com/golang/lint/golint
+	go mod tidy
+	#go get -u github.com/kisielk/errcheck
```

Comment:
These should probably say

    go install github.com/kisielk/errcheck

To bring them up to date!